04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,74 @@
+const Product = require('../models/product')
+
+const getAllProductsStatic = async(req, res)=>{
+   
+    const products = await Product.find({}).select('name price').limit(10)
+    res.status(200).json({products, nbHits: products.length})
+}
+
+const getAllProducts = async(req, res)=>{
+    const {featured, company, name, sort, fields, numericFilters} = req.query;
+    const queryObject = {}
+
+    if(featured){
+        queryObject.featured = featured === 'true'? true: false
+    }
+
+    if(company){
```

Comment:
This is great but just a tool for the toolbox here. I sometimes like to use the logical and operator for a quick conditional like this one

`company && queryObject.company = company` is the same result as what you have here for example